app/cypress/e2e/SecvisogramPage/FormEditorTab.cy.js

```diff
@@ -384,4 +384,43 @@ describe('SecvisogramPage / FormEditor Tab', function () {
       ).should('have.class', 'text-green-600')
     }
   })
+
+  it('prefills product IDs for branch/product', function () {
+    for (const user of getUsers()) {
+      cy.intercept(
+        '/.well-known/appspecific/de.bsi.secvisogram.json',
+        getLoginEnabledConfig()
+      ).as('wellKnownAppConfig')
+      cy.intercept(getLoginEnabledConfig().userInfoUrl, getUserInfo(user)).as(
+        'apiGetUserInfo'
+      )
+
+      cy.visit('?tab=EDITOR')
+      cy.wait('@wellKnownAppConfig')
+      cy.wait('@apiGetUserInfo')
+
+      // check if branch full product name is filled with a generated product ID
+      cy.get(
+        '[data-testid="menu_entry-/product_tree/branches-add_item_button"]'
+      ).click({ force: true })
+      cy.get(
+        '[data-testid="menu_entry-/product_tree/branches/0/product"]'
+      ).click()
+      cy.get(
+        '[data-testid="attribute-product_tree-branches-0-product-product_id"] input'
+      ).should('have.value', 'CSAFPID-0001')
+
```

Comment:
Add a test here for an item in `full_product_names` element.

Reply:
extended the test